lte/gateway/c/core/oai/tasks/nas/nas_proc.c

```diff
@@ -358,14 +359,15 @@ status_code_e nas_proc_authentication_info_answer(
   }
 
   if (!(emm_ctxt_p)) {
-    OAILOG_ERROR(
-        LOG_NAS_EMM, "That's embarrassing as we don't know this IMSI\n");
+    OAILOG_ERROR_UE(
+        LOG_NAS_EMM, emm_ctxt_p->_imsi64,
+        "That's embarrassing as we don't know this IMSI\n");
```

Comment:
if emm_ctxt_p is null, you cannot access it

Reply:
Resolved